src/test/shared/sam/cli/samCliInfo.test.ts

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import * as assert from 'assert'
+import { TestLogger } from '../../../../shared/loggerUtils'
+import { SamCliInfoInvocation, SamCliInfoResponse } from '../../../../shared/sam/cli/samCliInfo'
+
+describe('SamInfoCliCommand', async () => {
+
+    class TestSamCliInfoCommand extends SamCliInfoInvocation {
+        public convertOutput(text: string): SamCliInfoResponse | undefined {
+            return super.convertOutput(text)
+        }
+    }
+
+    let logger: TestLogger
+
+    before(async () => {
+        logger = await TestLogger.createTestLogger()
+    })
+
+    after(async () => {
+        await logger.cleanupLogger()
+    })
+
+    it('converts sam info response to SamCliInfoResponse', async () => {
+        const response: SamCliInfoResponse | undefined = new TestSamCliInfoCommand()
+            .convertOutput('{"version": "1.2.3"}')
```

Comment:
We should consult SAM CLI to see if they expect other fields to be added to the info response; if so, should we add some additional fields to ensure it's future-compatible?

Reply:
Other fields will indeed be added over time.
Ultimately, the sam --info output goes through this:
```
JSON.parse(text) as SamCliInfoResponse
```
* Additional fields would exist on the object but not be accessed by code using the `SamCliInfoResponse` object
* New fields that are mandatory require a toolkit update, and require a new minimum sam cli version

Reply:
I think it's reasonable to add a single test with a `version` and another unrelated property. It shouldn't matter to the current code, but the test will protect against regressions if we ever end up changing the underlying implementation.